x-pack/legacy/plugins/maps/common/descriptor_types.d.ts

```diff
@@ -89,7 +89,6 @@ export type XYZTMSSourceDescriptor = {
 };
 
 export type JoinDescriptor = {
-  leftField: string;
```

Comment:
Why was this removed? `leftField` is a member of the JoinDescriptor. You can not do a join without it.